cmd/airgap/listImages.go

```diff
@@ -13,35 +13,30 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
```

Comment:
I think it's better have underscore instead of camel case in filename. camelcase could led to problems on case insensitive fs (like macos)